system-addon/test/functional/mochitest/head.js

```diff
@@ -115,13 +113,9 @@ async function check_highlights_elements(selector, length, message) { // eslint-
   let browser = gBrowser.selectedBrowser;
   await waitForPreloaded(browser);
 
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", false);
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", true);
-
   await BrowserTestUtils.waitForCondition(() => content.document.querySelector(selector), `No elements matching ${selector} were found`);
 
-  let found = await ContentTask.spawn(browser, selector, arg =>
-    content.document.querySelectorAll(arg).length);
+  let found = content.document.querySelectorAll(selector).length;
```

Comment:
Where did you see this type of `content` usage? https://bugzilla.mozilla.org/show_bug.cgi?id=1127393 seems like it's saying this is a bad pattern ?

Reply:
Actually, this just happens to slip through the eslint `no-cpows-in-tests` rule because it's too complicated to test accurately whether `content` is in a content task.
https://searchfox.org/mozilla-central/source/tools/lint/docs/linters/eslint-plugin-mozilla.rst#no-cpows-in-tests

Reply:
I think a lot of tests use helper functions that spawn the ContentTask and I never got the difference. Will update.